weareinreach/inreach-catalog
# add COVID-19 sub category

## Description
add COVID-19 sub category to Medical search Filter

*Note this PR needs the matching PR for Control Panel, else there will be no service in the DB with this value
https://github.com/asylum-connect/control-panel/pull/85

## Asana ticket:
https://app.asana.com/0/1132189118126148/1201731585269564

## PR Checklist




## How to Test

1. using control panel (or manually via the DB) add COVID-19 services as a subtype under Medical for an organization
2. from the Catalog front end, Select COVID-19 service under Search Filter
3. expect the above org to be found
